### ISSUE TITLE
More user friendly handling of evaluator measurements

### ISSUE THREAD

**Issue by foxriver76** [scikit-multiflow/scikit-multiflow#54]
Wouldn't it be better to print the results at the end with 4 decimals like in the plot window? If I set "show_plot=false" I only get 3 decimals printed at the end of the evaluation because in the code it is written:

```python
 if 'performance' in self.metrics:
                logging.info('{} - Accuracy     : {:.3f}'.format(
                    self.model_names[i], self.global_classification_metrics[i].get_performance()))
```

Also I think it would be user friendly to have a method like evaluator.get_measurements which returns Kappa, Accuracy and so on for all models. Currently I have to use 

```python
evaluator.global_classification_metrics[0].get_performance()
```
for getting accuracy of the first model etc.

**Comment by foxriver76** [comment]
That seems to be a good solution. The main problem from my point was that there is currently no documented way to access the metrics.

**Comment by jacobmontiel** [comment]
> there is currently no documented way to access the metrics.

Actually, we haven't thought about this use case. But it certainly can be useful.

**Comment by jacobmontiel** [comment]
Changes are available in ba2f6f12c0777aad31f5874e9f89d2f5aea1ec70

Summary:

Evaluators and Metrics improvements
- Add methods to easily retrieve performance (mean, current) measurements from evaluators
- Rename measurement scope 'global' and 'sliding' for 'mean' and 'current' respectively.
- Now reported performance has the same precision in the plot and in the evaluation summary (log).
- Rename 'performance' to 'accuracy'.
- Create constants.py to keep constant values used across the package.
- Update examples.
- Update tests.
- Update documentation.
- Minor non-functional changes in visualizer and evaluators.
